guides/development/test-manual-installation-with-agents.md

```diff
@@ -0,0 +1,42 @@
+# Testing the setup
+
+In order to test the [manual installation locally](https://github.com/trento-project/docs/blob/main/guides/manual-installation.md) deploy an agent using the available `trento-agent` package. This package is already available in SLES 15 SP5, so we can install it using zypper:
+
+```bash
+zypper install trento-agent
+```
+
+## Configuring the Agent host with the Self-Signed Certificate
+
+**Step 1**: Copy the self signed certificate `trento.crt` from the trento-server to the agent machine. Use `scp` to transfer the certificate to `/etc/pki/trust/anchors/`.
+
+Example:
+
+```bash
+scp <<TRENTO_SERVER_MACHINE_USER>>@<<TRENTO_SERVER_MACHINE_IP>>:/etc/ssl/certs/trento.crt /etc/pki/trust/anchors/
+```
+
+**Step 2**: Update the certificate store:
+
+```bash
+update-ca-certificates
+```
+
+Configure Trento using the `/etc/trento/agent.yaml` file, and make sure to use `https` for the `server-url` parameter. Refer to https://documentation.suse.com/sles-sap/trento/html/SLES-SAP-trento/index.html#sec-trento-installing-trentoagent for more details.
```

Comment:
Shouldn't that be a step as well? It's an action that the user has to execute. 

Another question: are you talking about the agent or the server? The name of the config implies the agent, so I'd recommend this wording:

```suggestion
Configure the Trento agent using the `/etc/trento/agent.yaml` file. Make sure to use `https` for the `server-url` parameter. Refer to https://documentation.suse.com/sles-sap/trento/html/SLES-SAP-trento/index.html#sec-trento-installing-trentoagent for more details.
```